database/altsignaddr.go

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2020-2021 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package database
+
+import (
+	"errors"
+	"fmt"
+
+	bolt "go.etcd.io/bbolt"
+)
+
+// The keys used to store alternate signing addresses in the database.
+var (
+	altSignAddrK = []byte("altsig")
```

Comment:
How about the key too?

Reply:
Don't want to change the keys because it will break compatibility any existing databases, and tbh the values of the keys don't really matter at all.

The proper way to change a db key would be to write code which performs a database migration, and in this case I just don't think its worth it.

Reply:
That was my guess too, asked just in case. Thanks!